config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -319,4 +351,60 @@ periodics:
     - name: config
       configMap:
         name: label-config
-
+# cip-sanity runs daily, to make sure that the destination GCRs do not deviate
+# away from the intent of the manifest.
+- interval: 24h
+  max_concurrency: 1
+  # This name is the "job name", passed in as "--job=NAME" for mkpj.
+  name: cip-sanity
+  # Enable Pod Utilities. See https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md.
+  decorate: true
+  extra_refs:
+  # Because of Pod Utilities, we clone the below repo automatically, and, get
+  # dropped into /home/prow/go/src/github.com/kubernetes/k8s.io.
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: master
+  spec:
+    containers:
+    # TODO: Move the official cip image to a more serious location. But it's not
+    # a huge problem because we pin it by the digest.
+    - image: gcr.io/cip-demo-staging/cip@sha256:876bfc74d070f2eeb94625780b78e757834e1854ec30bc539cfb7553274905d9
+      command:
+      - multirun.sh
+      args:
+      # This is aided by Pod Utilities because of "decorate: true", which
+      # populates the manifest repo for us. There is also PULL_REFS and
+      # PULL_BASE_SHA defined for us.
+      #
+      # If you run mkpj for this postsubmit, the job will query you for the base
+      # ref to fetch (name of a branch, not a SHA, e.g. "master").
+      - /app/cip-docker-image.binary
+      - k8s-staging-cluster-api/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+      - k8s-staging-coredns/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+      - k8s-staging-csi/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+      # TODO: Uncomment this once
+      # https://github.com/GoogleCloudPlatform/k8s-container-image-promoter/issues/16
+      # lands.
+      #env:
+      #- name: CIP_OPTS
+      #  # Be aggressive about keeping the prod registry clean.
+      #  value: "-delete-extra-tags"
+      imagePullPolicy: Always
+      volumeMounts:
+      - name: k8s-gcr-prod-service-account-creds
+        mountPath: /etc/k8s-gcr-prod-service-account
+        readOnly: true
+    # Create a volume from a Kubernetes Secret. The Secret can be created as
+    # per: https://kubernetes.io/docs/concepts/configuration/secret/.
+    #
+    # In our case we downloaded a service account secret key file from GCP, then ran
+    #
+    #   kubectl create secret generic k8s-gcr-prod-service-account \
+    #     --from-file=service-account.json=PATH/TO/cip-demo-staging-e881274d0046.json
+    #
+    # and so our secretName is k8s-gcr-prod-service-account.
+    volumes:
+    - name: k8s-gcr-prod-service-account-creds
+      secret:
+        secretName: k8s-gcr-prod-service-account
```

Comment:
I forget, does this secret exist in the service cluster yet?

Reply:
Not yet --- can we create it after this PR is merged?

Reply:
Secret now exists